code/drasil-data/lib/Data/Drasil/Concepts/Documentation.hs

```diff
@@ -75,6 +75,8 @@ typUnc      = commonIdeaWithDict "typUnc"      (cn' "typical uncertainty")
 sec         = commonIdeaWithDict "section"     (cn' "section")                                       "Sec"     [documentc]
 srs         = commonIdeaWithDict "srs"         softReqSpec                                           "SRS"     [softEng]
 notebook    = commonIdeaWithDict "notebook"    (cn' "notebook")                                      "NB"      [softEng]
+refBy       = commonIdeaWithDict "refBy"       (cn "referenced by")                                  "RefBy"   [documentc]
```

Comment:
SUPER minor nitpick is to just add a space to help align this "dabatase" table
```suggestion
refBy       = commonIdeaWithDict "refBy"       (cn  "referenced by")                                 "RefBy"   [documentc]
```

Reply:
Thanks for catching! Fixed it now.